### PR TITLE
chore: update engines to 3.6.0-23.d5ba6c62f1d540b2a5e69a48e7fe55ba658635e6

### PR DESCRIPTION
**Pull request by aqrln**
Ref https://github.com/prisma/prisma-engines/pull/2418
Ref https://github.com/prisma/prisma-engines/pull/2443
Ref https://github.com/prisma/prisma/issues/8854

### CODE REVIEW

File: packages/cli/package.json [hunk 1]
```diff
@@ -67,9 +67,9 @@
   "devDependencies": {
     "@prisma/client": "workspace:*",
     "@prisma/debug": "workspace:*",
-    "@prisma/fetch-engine": "3.6.0-19.9372e13dbcc967fb9bd1f954bed16d7b7b96d4f9",
+    "@prisma/fetch-engine": "3.6.0-20.query-engine-in-filter-03d89ba07c179d4a935e06dc5b74c78a24de3164",
```
Comment (janpio) [inline packages/cli/package.json:70]: This looks like a special engine build (integrat, that should probably not make it into `main`, right?

Comment (aqrln) [reply packages/cli/package.json:70]: Yes, once the engines PR is merged, we'll replace it with the proper build before merging this.

Comment (aqrln) [reply packages/cli/package.json:70]: https://github.com/prisma/prisma-engines/pull/2443 is merged, waiting for the binaries to be built and released

**Review by janpio — APPROVED** [review]

File: packages/cli/package.json (hunk 1 above)
Comment (Jolg42) [reply packages/cli/package.json:70]: Looks test like pipeline is stuck and this engine is not yet out from https://buildkite.com/prisma/test-prisma-engines

Comment (aqrln) [reply packages/cli/package.json:70]: @Jolg42 should be unblocked soon: https://buildkite.com/prisma/test-prisma-engines/builds/11301

**Review by Jolg42 — APPROVED** [review]

**Pull request merged**